ghost/core/core/server/services/mentions/BookshelfMentionRepository.js

```diff
@@ -63,7 +63,9 @@ module.exports = class BookshelfMentionRepository {
      * @returns {Promise<Page<import('@tryghost/webmentions/lib/Mention')>>}
      */
     async getPage(options) {
-        const page = await this.#MentionModel.findPage(options);
+        // TODO: sqlite uses 0, mysql uses false... how do we construct this filter properly?
+        const filter = `deleted:[0,false]` + (options.filter ? `+(${options.filter})` : '');
```

Comment:
Not sure how we should handle `sqlite` using `0/1` vs. mysql using `true/false`.

Reply:
`true/false` _should_ work for SQLite! I believe knex will handle the difference

Reply:
I've updated this to use the `enforcedFilters` method in models - which is used as the `overrides` property in `@tryghost/nql`